src/kakarot/kakarot.cairo

```diff
@@ -438,3 +447,22 @@ func eth_send_transaction{
 
     return result;
 }
+
+@l1_handler
+func handle_l1_message{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}(from_address: felt, l1_sender: felt, to_address: felt, value: felt, data_len: felt, data: felt*) {
+    alloc_locals;
+    let (is_authorized) = Kakarot_authorized_message_senders.read(from_address);
```

Comment:
should we create a `Kakarot.get_authorized_message_sender` ? it's a bit weird to use the storage var directly to read but not to write